Header header
geometry_msgs/PoseStamped[] poseArray